<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lens</class>
 <widget class="QDialog" name="Lens">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>744</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AvisionR -Lens</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <widget class="QTableView" name="tableView_lens">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>37</y>
     <width>726</width>
     <height>347</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;

}

QTableCornerButton::section 
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;
}

color:rgb(255, 255, 255);
background-color: #262626;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AddOns">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>13</y>
     <width>311</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #686868;</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_Search">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>390</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>91</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_Clear">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>390</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>91</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_New">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>390</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>81</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Type">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>13</y>
     <width>268</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #686868;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Lens">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>13</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #686868;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_Done">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>390</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>91</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_AddOns</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Lens</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Type</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
